frontend/store/collaterals.ts

```diff
@@ -98,12 +100,17 @@ export const actions = {
                     priceDropRatio: error.toString(),
                 };
             });
+            const autoRouteData = await getUniswapAutoRoute(network, collateral.symbol);
+            const autoRouteExchanges = autoRouteData.route[0].tokenPath.map(p => p.symbol);
+
             const updated = {
                 ilk: collateral.ilk,
                 marketUnitPrice,
                 secondsBetweenPriceDrops: calcParameters.secondsBetweenPriceDrops,
                 priceDropRatio: calcParameters.priceDropRatio,
                 tokenAddress,
+                autoRouteQuote: new BigNumber(autoRouteData.quote.toFixed(2)),
```

Comment:
Why do you turn it into a `BigNumber` in order to turn it back to string with the worse method (`toString`) in the template? Suggestion: keep it as string

Reply:
I had done this to be able to determine if the value provided is a valid quote number (BigNumber) or an error message (string).

This way I can add the popup as well to display full error messages.